java/bulk-import/bulk-import-runner/docker/emrs/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022-2023 Crown Copyright
```

Comment:
I think it would be clearer if this file was in a directory called "ear-serverless"

Reply:
It has been renamed emr-serevrless